recipes-extended/xen/xen-common.inc

```diff
@@ -102,32 +122,13 @@ SRC_URI_append = " \
     file://ocaml-makefiles-sysroot.patch \
 "
 
-COMPATIBLE_HOST = 'i686-oe-linux|(x86_64.*).*-linux|aarch64.*-linux'
```

Comment:
No, I am not building on ARM. That is an entertaining idea though.
I'm removing this line from the OpenXT recipe since I don't think there's a need for OpenXT's Xen recipes to explicitly diverge from the meta-virtualization definitions of COMPATIBLE_HOST.

Reply:
> That is an entertaining idea though.

+1!

Agreed with not diverging, thanks!